src/components/ItemList/createItemList.js

```diff
@@ -0,0 +1,14 @@
+import constants from './constants';
+import './itemList.scss';
```

Comment:
Try to group the imports by category. In this case for instance, the `.scss` import should go at the end.